src/main/java/org/openpnp/gui/JobPlacementsPanel.java

```diff
@@ -72,7 +75,8 @@
     private static Color statusColorReady = new Color(157, 255, 168);
     private static Color statusColorError = new Color(255, 157, 157);
 
-    public JobPlacementsPanel(JobPanel jobPanel) {
+    public JobPlacementsPanel(JFrame frame, JobPanel jobPanel) {
+    	this.frame = frame;
```

Comment:
I've been moving to MainFrame.get() rather than passing the frame around everywhere. You can also use getTopLevelAncestor() in UI classes. No need to change this, just keep in mind for the future.